release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -429,6 +429,13 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
 
 In {product-title} {product-version}, the installation program uses the Cluster API instead of Terraform to provision cluster infrastructure during installations on {ibm-cloud-title}.
 
+[id="ocp-4-19-GCP-custom-dns_{context}"]
+==== Installing a cluster on {gcp-first} with user-provisioned DNS
+
+With this release, you can enable user-provisioned DNS instead of the default cluster-provisioned DNS solution. For example, your organization's security policies might not allow the use of public DNS services such as Google Cloud DNS. In such scenarios, you can use your own DNS service to bypass the public DNS service. You can manage your DNS only for the IP addresses of the API and Ingress servers. If you use this feature, you must provide your own DNS solution that includes records for `api.<cluster_name>.<base_domain>.` and `*.apps.<cluster_name>.<base_domain>.`.
```

Comment:
```suggestion
With this release, you can enable user-provisioned DNS instead of the default cluster-provisioned DNS solution. For example, your organization's security policies might not allow the use of public DNS services such as Google Cloud DNS. You can manage your DNS only for the IP addresses of the API and Ingress servers. If you use this feature, you must provide your own DNS solution that includes records for `api.<cluster_name>.<base_domain>.` and `*.apps.<cluster_name>.<base_domain>.`.
```
removed a bit of redundancy, otherwise LGTM.